cfgov/core/views.py

```diff
@@ -123,3 +126,20 @@ def submit_comment(data):
                              })
 
     return response
+
+
+@csrf_exempt
```

Comment:
@willbarton @rosskarchner i think you guys are right.  it's only a single endpoint, and i forgot that djangorestframework also makes you use a decorator as well as importing the module.  it was just a suggestion - in case that single endpoint becomes more in the future.